src/m365/spo/commands/file/file-checkin.ts

```diff
@@ -132,7 +137,8 @@ class SpoFileCheckinCommand extends SpoCommand {
     }
 
     if (args.options.url) {
-      requestUrl = `${args.options.webUrl}/_api/web/GetFileByServerRelativeUrl('${formatting.encodeQueryParameter(args.options.url)}')/checkin(comment='${comment}',checkintype=${type})`;
+      const siteRelUrl = urlUtil.getServerRelativePath(args.options.webUrl, args.options.url);
```

Comment:
I find this naming confusing: we're calling a method that's named `getServerRelativePath` and assign the value it returns to `siteRelUrl` (server-relative vs. site-relative). Let's either adjust the name or add a comment to clarify what we're doing so that it's obvious without having to step through the code.